public/app/features/explore/LiveLogs.tsx

```diff
@@ -73,25 +72,6 @@ class LiveLogs extends PureComponent<Props, State> {
     };
   }
 
-  componentDidUpdate(prevProps: Props) {
```

Comment:
So what this did was to allow user to pause the live view by just scrolling the view up. Rationale was if you want to focus on some line then you don't wont that line to jump up all the time and you have to chase it. Because we were also scrolling the view to the bottom line every time when not paused, this was needed to figure out if user scrolled up, or if new lines appeared and that is why the view scroll position is not at the bottom.

Reply:
I have a feeling the issue is I did not though there would be 2 scroll views. I though when live tailing the page would have fixed 100% height and only the internal log view would be scrolling

Reply:
Yeah, when I was playing with live tailing that was always the case as I did just 1 query that was 1 line-long. However, if you have more queries/long queries, you get the second scroll area. And we can't remove it, cause if you hae multiple queries + small display, you would end up with very narrow live tailing window.

But anyway, I have tested my solution and it still works in a way where if you scroll, it pause the live tailing.

Reply:
Hmm you are right. Don't like thinking that this elaborate code actually did not do anything substantial but seems to be the case :D